frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "@headlessui/react": "^1.7.3",
     "@heroicons/react": "^2.0.12",
     "@tailwindcss/aspect-ratio": "^0.4.2",
+    "bootstrap": "^5.2.2",
```

Comment:
We initially decided to use tailwind CSS and not bootstrap. So for a more consistent design, it would be great if you could use tailwind instead of bootstrap.